UPGRADE.md

```diff
@@ -1,46 +1,130 @@
+This document guides you through the process of upgrading Kong. First, check if
+a section named "Upgrade to Kong `x.x.x`" exists (`x.x.x`) being the version
+you are planning to upgrade to. If such a section does not exist, the upgrade
+you want to perform does not have any particular instructions, and you can
+simply consult the [Suggested upgrade path](#suggested-upgrade-path).
 
 ## Suggested upgrade path
 
-Unless indicated otherwise in one of the upgrade paths of this document, it is possible to upgrade Kong **without downtime**:
+Unless indicated otherwise in one of the upgrade paths of this document, it is
+possible to upgrade Kong **without downtime**:
 
-Considering that Kong is already running on your system, acquire the latest version from any of the available [installation methods](https://getkong.org/install/) and proceed to installing it, overriding your previous installation. Once done, consider that this is a good time to also modify your configuration.
+Considering that Kong is already running on your system, acquire the latest
+version from any of the available [installation
+methods](https://getkong.org/install/) and proceed to installing it, overriding
+your previous installation. Once done, consider that this is a good time to
+also modify your configuration.
 
 Then, run any new migration to upgrade your database schema:
 
 ```shell
 $ kong migrations up [-c configuration_file]
-...
-[OK] Schema up to date
 ```
 
-If you see the "Schema up to date" message, you only have to [reload](https://getkong.org/docs/latest/cli/#reload) Kong:
+If the command is successful and no migration has been ran
+(no output), then you only have to
+[reload](https://getkong.org/docs/latest/cli/#reload) Kong:
 
 ```shell
 $ kong reload [-c configuration_file]
 ```
 
-**Reminder**: `kong reload` leverages the Nginx `reload` signal and seamlessly starts new workers taking over the old ones until they all have been terminated. This will guarantee you no drop in your current incoming traffic.
+**Reminder**: `kong reload` leverages the Nginx `reload` signal and seamlessly
+starts new workers taking over the old ones until they all have been
+terminated. This will guarantee you no drop in your current incoming traffic.
+
+## Upgrade to `0.10.x`
+
+Due to the breaking changes introduced in this version, we recommend that you
+careful test your cluster deployment.
+
+Kong 0.10 introduced the following breaking changes:
+
+- API Objects (as configured via the Admin API) do **not** support the
+  `request_host` and `request_uri` fields anymore. The 0.10 migrations should
+  upgrade your current API Objects, but make sure to read the new [0.10 Proxy
+  Guide](https://getkong.org/docs/0.10.x/proxy) to learn the new routing
+  capabilities of Kong. On the good side, this means that Kong can now route
```

Comment:
Or just remove it "This means that Kong can now..."

Less words is typically better, even if it means taking out "friendly" things like this - IMHO.